### DISCUSSION TITLE
Next go-git version ?

### DISCUSSION THREAD

**Discussion by BeChris** [go-git/go-git#1253] [General]
Hello,
Do you plan to deliver a v5.13 within next weeks ?
Or is v6 the next version to be delivered ?

**Comment by pjbgf** [comment]
@BeChris unfortunately I am not sure when we will be cutting the next release. The current recommendation is for users to bump based on the latest version from `master` or from `v6-exp`, based on whether they are on v5 or are keen on the new features that v6 will contain.

**Comment by BeChris** [comment]
OK, many thanks for your answer.
I will stay on master branch until a new v5.xx is released.

**Comment by Ravikc** [comment]
Hello @pjbgf , when is the plan to release v6? We are looking to make use of [this](https://github.com/go-git/go-git/pull/1204) feature for our Azure DevOps git operations.

**Reply by pjbgf** [reply]
@Ravikc there is no official timeline for the `v6` release, as the plan is to land `sha256` and update all the interfaces before an official v6 version takes place - so I believe this will take a while. I would recommend using the latest from `v6-exp` with `go get github.com/go-git/go-git@v6-exp`. 

Now if you prefer using released versions, you could use `v5` with [a workaround](https://github.com/go-git/go-git/blob/master/_examples/azure_devops/main.go).

**Reply by Ravikc** [reply]
Thanks @pjbgf, is the v6-exp branch stable enough to be used in production? Ofcourse we will test our use-case before using it, but just wanted to get your recommendation on it.

**Reply by pjbgf** [reply]
@Ravikc that's the aim really, to keep `v6-exp` as stable as possible, from a functionality perspective - the tests in place should assure that. Just beware that it is bound to have several API changes.
